app/models/requests/psr/MemberPayments.scala

```diff
@@ -75,7 +75,11 @@ case class MemberLumpSumReceived(
   lumpSumAmount: Double,
   designatedPensionAmount: Double
 ) {
-  val zero: Boolean = lumpSumAmount == 0.0 && designatedPensionAmount == 0.0
+  val zero: Boolean = lumpSumAmount == 0d && designatedPensionAmount == 0d
```

Comment:
Would it be useful to differentiate in naming between this `zero` and the `zero` below? I.e. rename this to `isZero` maybe?